sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -121,11 +122,12 @@ def test_ndframe_tests(self):
             'pandas.core.generic.NDFrame.convert_dtypes': ['*'],
             'pandas.core.generic.NDFrame.copy': ['*'],
             'pandas.core.generic.NDFrame.droplevel': ['*'],
+            'pandas.core.generic.NDFrame.get': ['*'],
             'pandas.core.generic.NDFrame.rank': [
                 # Modified dataframe
                 'df'
             ],
-            'pandas.core.generic.NDFrame.rename': [
+            'pandas.core.generic.NDFrame._rename': [
```

Comment:
The `rename` [function](https://github.com/pandas-dev/pandas/blame/ea2b0fdc64d2a7d28b5e622d9617d7236374fbbe/pandas/core/frame.py#L5092) got renamed (no pun intended 😄 )to `_rename`